pygments/lexers/jslt.py

```diff
@@ -0,0 +1,95 @@
+"""
+    pygments.lexers.jslt
+    ~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for the JSLT language
+
+    :copyright: Copyright 2021 by the Pygments team, see AUTHORS
+    :license: BSD, see LICENSE for details
+"""
+
+from pygments.lexer import RegexLexer, combined, words
+from pygments.token import Comment, Error, Keyword, Name, Number, Operator, \
+    Punctuation, String, Whitespace
+
+
+__all__ = ['JSLTLexer']
+
+
+_WORD_END = r'(?=[^0-9A-Z_a-z-])'
+
+
+class JSLTLexer(RegexLexer):
+    """
+    For `JSLT <https://github.com/schibsted/jslt>`_ source.
+
+    .. versionadded:: 2.10.0
```

Comment:
Please change to `2.10`

Reply:
Addressed in https://github.com/pygments/pygments/pull/1821/commits/30fff5120c0ff842e79f4ed074117148b04e6253.